store/tikv/region_cache.go

```diff
@@ -709,8 +777,217 @@ func (r *Region) ContainsByEnd(key []byte) bool {
 		(bytes.Compare(key, r.meta.GetEndKey()) <= 0 || len(r.meta.GetEndKey()) == 0)
 }
 
-// Store contains a tikv server's address.
+// fn loads the Store info given by storeId.
+type resolveFunc func(ctx context.Context, id uint64) (*metapb.Store, error)
+
+// Store contains a kv process's address.
 type Store struct {
-	ID   uint64
-	Addr string
+	addr  atomic.Value // loaded store address(*string)
+	peer  *metapb.Peer // store's peer
```

Comment:
What does `store's peer` mean? A store has many peers, one for each region.

Reply:
oh...that's a bug...fixed